newIDE/app/src/ProjectCreation/LocalStarters.js

```diff
@@ -232,6 +232,12 @@ export default class LocalStarters extends Component<Props, State> {
                     onClick={() => this.createFromExample(i18n, 'pairs')}
                   />
                   <ListItem
+                    leftIcon={
+                      <ListIcon
+                        iconSize={40}
+                        src="res/starters_icons/new.png"
+                      />
+                    }
```

Comment:
Can you post a screenshot of what the result looks like? Thanks :)

Reply:
### 
![newimage](https://user-images.githubusercontent.com/53819287/94897122-ecce3d80-0486-11eb-8bee-436db858a64f.PNG)